django/db/models/expressions.py

```diff
@@ -1068,6 +1068,11 @@ def as_sql(self, compiler, connection, template=None, case_joiner=None, **extra_
             sql = connection.ops.unification_cast_sql(self.output_field) % sql
         return sql, sql_params
 
+    def get_group_by_cols(self, alias=None):
+        if not self.cases and isinstance(self.default, Value):
+            return []
```

Comment:
You can delegate it to `self.default`

```suggestion
        if not self.cases:
            return self.default.get_group_by_cols(alias)
```

Reply:
Much better, thanks!